FormatSchemes/ProjectFile/ShowFile_v0.xsd

```diff
@@ -70,6 +70,11 @@ They can be used to implement UI controls on the touch screens besides scene swi
     		<annotation>
     			<documentation>The ID is used to query this filter during scene construction (to the filters output to the inputs of other filters). It must not contain colons.</documentation>
             </annotation>
+        </attribute>
+		<attribute name="pos" type="string" use="required">
+    		<annotation>
+    			<documentation>Filter node position inside the editor ui.</documentation>
+            </annotation>
```

Comment:
We'll add this now: For the future: If you need to store some information, you might as well use the general purpose key value storage within show files.